<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockFft</class>
 <widget class="QDockWidget" name="DockFft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/icons/signal.svg</normaloff>:/icons/icons/signal.svg</iconset>
  </property>
  <property name="toolTip">
   <string>FFT performance and display settings</string>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>FFT Settings</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="fftSettingsVLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <layout class="QFormLayout" name="fftSettingsFormLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="fftSizeLabel">
        <property name="text">
         <string>FFT size:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="fftSizeComboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;Number of FFT points to calculate. Higher values will require more CPU time. This will not influence the number of points on the display since that parameter is adjusted automatically according to the display size.
&lt;/html&gt;</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAlphabetically</enum>
        </property>
        <item>
         <property name="text">
          <string>32768</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16384</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8192</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4096</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3840</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2048</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1024</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>768</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>512</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fftRateLabel">
        <property name="text">
         <string>Rate:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="fftRateComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>FFT refresh rate.</string>
        </property>
        <property name="currentIndex">
         <number>5</number>
        </property>
        <item>
         <property name="text">
          <string>30 fps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25 fps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20 fps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17 fps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15 fps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 fps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 fps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 fps</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fftAvgLabel">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;FFT averaging speed.&lt;/p&gt;&lt;p&gt;Also known as video filter on spectrum analyzers.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Averaging:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="fftAvgSlider">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;FFT averaging speed.&lt;/p&gt;&lt;p&gt;The averaging does not apply to the waterfall, which is always updated with the latest unfiltered samples.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="fftSettingsSplitHLayout">
      <item>
       <widget class="QLabel" name="wfLabel">
        <property name="text">
         <string>Pand</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="fftSplitSlider">
        <property name="toolTip">
         <string>Spatial distribution between pandapter and waterfall</string>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pandLabel">
        <property name="text">
         <string>WF</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
